### PR TITLE
Modify home page

### PR DESCRIPTION
**Pull request by MaorW**
Fix issue #159 
Please merge it after PR #167  of Noam

### CODE REVIEW

**Review by lmilbaum — DISMISSED** [review]
LGTM

**Comment by MaorW** [comment]
Conflicts fixed

**Review by MaorW — COMMENTED** [review]
Hey, I've fixed my conflicts

File: templates/home.html [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-        Sender Name: <input type="text" name="username"><br>
-        <p id="time">
-            <script>
-                document.getElementById("time").innerHTML = Date();
-
-            </script><br>
-        </p>
-        <input type="submit" value="Send Message"> <input type="Reset" name="reset1" value="Reset">
-    </form><br>
+{% extends "layout.html" %}
```
Comment (MaorW) [inline templates/home.html:1]: Fixed conflicts here

**Review by MaorW — COMMENTED** [review]
Hey, I rereview my fixed conflicts

**Review by lmilbaum — APPROVED** [review]
LGTM

**Pull request merged**